sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -26,7 +27,9 @@
      * Creates an instance of IdentityClientOptions with default settings.
      */
     public IdentityClientOptions() {
-        authorityHost = DEFAULT_AUTHORITY_HOST;
+        Configuration configuration = Configuration.getGlobalConfiguration().clone();
```

Comment:
Is there a need to have a short lived clone of the global configuration? If anything it may be better to use it directly as it could load into static memory the environment setting and will be accessed quicker in the future.

Reply:
I followed they way in SharedTokenCacheCredential，it cloned a configuration in its constructor.

Reply:
Removed clone.